docker/services/uwsgi.d/60_processes.ini

```diff
@@ -0,0 +1,4 @@
+[uwsgi]
+# Default number of processes and threads for the app
+processes = 3
```

Comment:
Should this be something like (no of cores - [some number of free cores])
https://uwsgi-docs.readthedocs.io/en/latest/Changelog-1.9.20.html#simple-math-in-configuration-files

Reply:
We run many instances of the circulation manager codebase on a single server, since each circulation manager can only handle a subset of the libraries we run currently.